source/_integrations/unifiprotect.markdown

```diff
@@ -161,3 +164,7 @@ The default settings on the stream integration will give you a 5-15+ second dela
 ### Cannot Play Audio to Speakers
 
 Unlike with many other things, playing audio to your speakers requires your Home Assistant to be able to reach your camera directly. Specifically via port `tcp/7004`. You can [enable debug logging](#enabling-debug-logging) and it will output the full FFmpeg command that will be run and the output from FFmpeg to help you troubleshoot why audio is not playing to the device.
+
+### Liveview Options for Viewport Missing Options or Out of Date
+
+Main control selects cannot have dynamic options since the options are exported out to voice assistants. After you add / remove / change a Liveview in UniFi Protect, you must restart Home Assistant to get the new options for your Viewport.
```

Comment:
```suggestion
Main control selects cannot have dynamic options since the options are exported out to voice assistants. After you add/remove/change a Liveview in UniFi Protect, you must restart Home Assistant to get the new options for your Viewport.
```

Grammar checker said to remove the ` ` between the slashes

Reply:
Anything that is marked as a config entity shouldn't be exported to voice assistants by default (except for some cases in HomeKit) so we should be able to refactor to allow dynamic updates in a future PR (not today)